stories/dooboo-ui/Select.stories.tsx

```diff
@@ -0,0 +1,283 @@
+import { Image, View } from 'react-native';
+import { Item, Select } from '../../main/Select';
+import React, { ReactElement } from 'react';
+
+import { ContainerDeco } from '../../storybook/decorators';
+import LoadingIndicator from '../../main/LoadingIndicator';
+import { storiesOf } from '@storybook/react-native';
+import styled from 'styled-components/native';
+
+const CustomContainer = styled.View` 
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+`;
+const TogglerWrapper = styled.View`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 10px;
+`;
+const ThemeSwitch = styled.Switch`
+  width: 30px;
+  height: 30px;
+`;
+const CustomTextInput = styled.TextInput`
+  background-color: #ffffff;
+  width: 190px;
+  height: 50px;
+  padding: 10px;
+  font-size: 12px;
+`;
+const CustomSelectContainer = styled.View`
+  width: 185px;
+  height: 45px;
+  align-items: center;
+  justify-content: center;
+`;
+const CustomText = styled.Text`
+  font-size: 10px;
+  font-weight: bold;
+  margin: 5px;
+`;
+
+const Default = (): React.ReactElement => {
+  // [Select] states
+  const [selectedValue, onSelectValue] = React.useState<
+    string | number
+  >();
+  const [isOpen, toggleSelect] = React.useState<
+    boolean
+  >(false);
+
+  return (
+    <CustomContainer style={{ justifyContent: 'center' }}>
+      <CustomSelectContainer>
+        <Select
+          open={isOpen}
```

Comment:
Open sounds like an action. How about `opened` here?

Reply:
`opened` sounds better! I'll fix that